### DISCUSSION TITLE
NOt an issue just a query

### DISCUSSION THREAD

**Discussion by lohtse** [armel/uv-k5-chirp-driver#19] [Announcements]
Love the FW and driver... 

I am using the feature update 3, So am curious as to when an updated driver will be release as the power settings has now changed and you added some excellent options. But they have changes from what ever set originally too low2 on radio.. Changing on the radio doesn't stay permanent but changing via driver and uploading too radio would.

So just curious as to when a possible updated driver will be available...

Amazing work on FW and driver...Thank you buddy

**Comment by armel** [comment]
Note that the power setting is one of the attributes of the channel. So if you change it in the Power menu (02/64), you need to save the channel again via the ChSave menu (16/64) if you want it to persist...

I'll be publishing v3.1 in a while. Of course, the Chirp driver is already ready. 

Armel F4HWN.

**Comment by lohtse** [comment]
> Note that the power setting is one of the attributes of the channel. So if you change it in the Power menu (02/64), you need to save the channel again via the ChSave menu (16/64) if you want it to persist...
> 
> I'll be publishing v3.1 in a while. Of course, the Chirp driver is already ready.
> 
> Armel F4HWN.

Thank you for a quick response.

I will wait until the updated driver is available as I need to change every channel(126 of them lol) as they all set too low2 due to the changes you introduced... I have always applied power settings using chirp too as is quicker for me..


Not a huge issue at the moment so can wait.. hence why I ask when a possible update might happen...
